code/game/objects/effects/effect_system/effects_smoke.dm

```diff
@@ -16,13 +16,12 @@
 	var/amount = 4
 	var/lifetime = 5
 	var/opaque = 1 //whether the smoke can block the view when in enough amount
+	use_fade = TRUE
 
-/obj/effect/particle_effect/smoke/New()
-	..()
-	alpha = 0
+/obj/effect/particle_effect/smoke/Initialize()
 	create_reagents(500)
-	animate(src, alpha = 255, time = 5)
 	START_PROCESSING(SSobj, src)
+	return ..()
```

Comment:
. = ..() like all other initalizes

Reply:
No, because that's why this whole thing needed a fix to start with.